jest/mockModal.js

```diff
@@ -13,11 +13,10 @@
 'use strict';
 
 const React = require('react');
-const Modal = require('../Libraries/Modal/Modal');
 
 function mockModal(BaseComponent: $FlowFixMe) {
   class ModalMock extends BaseComponent {
-    render(): React.Element<typeof Modal> {
```

Comment:
Can we add a comment here on why this is FlowFix'ed and maybe an issue to investigate?

Reply:
@yungsters suggested a solution without the $FlowFixMe so this is resolved